corehq/tabs/tabclasses.py

```diff
@@ -907,10 +906,11 @@ def title(self):
 
     @classmethod
     def make_app_title(cls, app):
-        return mark_safe("%s%s" % (
-            escape(strip_tags(app.name)) or '(Untitled)',
+        return format_html(
+            '{}{}',
+            strip_tags(app.name) or '(Untitled)',
```

Comment:
possible to mark "Untitled" for translation? :)

Reply:
[`159e9de` (#29364)](https://github.com/dimagi/commcare-hq/pull/29364/commits/159e9dead2f1aad5678b6e8ffefedcd57653f76a)